src/software/backend/BUILD

```
@@ -65,8 +65,20 @@ cc_library(
 
 cc_library(
     name = "robot_status",
+    srcs = ["robot_status.cpp"],
     hdrs = ["robot_status.h"],
-    deps = [],
+    deps = [
+        "//software/proto:sensor_msg_cc_proto",
```

```suggestion
        "//shared/proto:tbots_msg_cc_proto",
```
Not sure about the exact spelling..

---

The one I could find under `shared/proto/BUILD` was `tbots_cc_proto`.